tests/components/mikrotik/test_hub.py

```diff
@@ -0,0 +1,29 @@
+"""The tests for the Mikrotik device tracker platform."""
+from __future__ import annotations
+
+from homeassistant.components import mikrotik
+from homeassistant.core import HomeAssistant
+
+from . import setup_mikrotik_entry
+
+
+async def test_info_routerboard(hass: HomeAssistant) -> None:
```

Comment:
In my opinion, these tests are missing the point as you mock `MikrotikData`, including the changes made in this PR.

I suggest you mock the API instead, so you can test if your changes work as expected.

Reply:
I've totally reworked the test to focus on testing the new behaviour of MikrotikData.
The test is much simpler to setup and requires the strict minimum mocking.
Is it better now?